lib/Authentication.php

```diff
@@ -149,16 +149,16 @@ public function signRequest(Psr7\Request $request, ?string $scope = null): Psr7\
         }
         $accessToken = $credsForAccessToken->getSecurityToken();
 
-        $relevantCreds = $this->awsCredentials;
+        $relevantCreds = $scope === null ? $this->awsCredentials : $this->grantlessAwsCredentials;
         if ($this->roleArn !== null) {
             if ($this->roleCredentials === null || $this->roleCredentials->expiresSoon()) {
                 $client = new StsClient([
                     'sts_regional_endpoints' => 'regional',
                     'region' => $this->region,
                     'version' => '2011-06-15',
                     'credentials' => [
-                        'key' => $relevantCreds->getAccessKeyId(),
-                        'secret' => $relevantCreds->getSecretKey(),
+                      'key' => $this->awsKey,
+                      'secret' => $this->awsSecret,
```

Comment:
Can you please fix the indentation here to match the 4-space indentation in the rest of the file? Thanks!

Reply:
Resolved on e42c66eb7471546f147374243da29d5f92f3e10b